addons/point_of_sale/static/src/app/models/pos_order.js

```diff
@@ -664,13 +690,9 @@ export class PosOrder extends Base {
             this.lines.reduce((sum, orderLine) => {
                 if (!ignored_product_ids.includes(orderLine.product_id.id)) {
                     sum +=
-                        orderLine.getUnitDisplayPriceBeforeDiscount() *
-                        (orderLine.getDiscount() / 100) *
-                        orderLine.getQuantity();
-                    if (
-                        orderLine.displayDiscountPolicy() === "without_discount" &&
-                        !(orderLine.price_type === "manual")
-                    ) {
+                        orderLine.getAllPrices().priceWithTaxBeforeDiscount -
+                        orderLine.getAllPrices().priceWithTax;
+                    if (orderLine.displayDiscountPolicy() === "without_discount") {
```

Comment:
@adgu-odoo Can you rereview this change? @sbel-odoo made a fix (https://github.com/odoo/odoo/pull/191448/files) around this code and now a test is failing (but only if watched, not during normal test running).